client/ayon_core/tools/workfiles/models/workfiles.py

```diff
@@ -587,6 +588,7 @@ def _create_workfile_info_entity(self, task_id, rootless_path, note):
 
         username = self._get_current_username()
         workfile_info = {
+            "id": uuid.uuid4().hex,
```

Comment:
What does this resolve? Wouldn't the server solve this for us? Or do we need the `id` in our stored workfile info somewhere so we can update it easily without requiring to query again?

Reply:
> Or do we need the id in our stored workfile info somewhere so we can update it easily without requiring to query again?


Yep, so we can update.